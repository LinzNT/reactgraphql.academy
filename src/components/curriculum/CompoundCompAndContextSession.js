import React from 'react'
import Ul, { Li } from '../layout/Ul'
import Session from './Session'

const CompoundCompAndContextSession = ({ hideTitle }) => (
  <Session
    hideTitle={hideTitle}
    title="Advanced React patterns to create even more reusable
    UIs"
  >
    <Ul>
      <Li>Compound Components, dynamically flow data between components</Li>
      <Li>Patterns and use cases using "context"</Li>
    </Ul>
  </Session>
)

export default CompoundCompAndContextSession
