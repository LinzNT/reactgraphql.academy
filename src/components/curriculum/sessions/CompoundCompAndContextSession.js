import React from 'react'
import Ul, { Li } from '../../layout/Ul'
import Session from './Session'

const CompoundCompAndContextSession = ({ title }) => (
  <Session title={title}>
    <Ul>
      <Li>Compound Components, dynamically flow data between components</Li>
      <Li>Patterns and use cases using "context"</Li>
    </Ul>
  </Session>
)

export default CompoundCompAndContextSession
