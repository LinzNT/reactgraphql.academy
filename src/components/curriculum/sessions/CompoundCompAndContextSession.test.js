import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import CompoundCompAndContextSession from './CompoundCompAndContextSession'

describe('<CompoundCompAndContextSession />', () => {
  it('should render component', () => {
    const wrapper = shallow(<CompoundCompAndContextSession />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
