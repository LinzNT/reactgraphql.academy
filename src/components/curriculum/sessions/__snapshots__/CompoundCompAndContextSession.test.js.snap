// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CompoundCompAndContextSession /> should render component 1`] = `
<Session>
  <styled.ul>
    <styled.li>
      Compound Components, dynamically flow data between components
    </styled.li>
    <styled.li>
      Patterns and use cases using "context"
    </styled.li>
  </styled.ul>
</Session>
`;
